content/barnin.de.md

```diff
@@ -5,13 +5,10 @@ slug: "barnin"
 ---
 
 ### Unterkunft
```

Comment:
Wahrscheinlich können wir Barnin auch erst einmal raus lassen, bis es (oder etwas anderes) wirklich wieder stattfindet.

Reply:
Habe die Seite herausgenommen. Auf der Seite Veranstaltungen habe ich den Eintrag (mit dem Hinweis, dass es aktuell pausiert) noch nicht gelöscht, da es gute Werbung für uns ist und wir uns als Verein mal Gedanken machen sollten, dass ein Summer Camp in Zukunft wieder angeboten wird.